cg-spec/editors_draft.html

```diff
@@ -153,7 +153,7 @@
 </head>
 <body>
   <section id="abstract">
-    <p>TODO</p>
+    <p>The Resource Description Framework (RDF) is a general-purpose framework for representing information on the Web. RDF-star extends RDF with a convenient way to make statements about other statements. This specification defines the abstract syntax of RDF-star as an extension of RDF's, it extends a number of RDF concrete syntaxes to support the new abstract syntax, as well as RDF's formal semantics. This specification also extends the SPARQL language to allow querying and updating of RDF-star data.</p>
```

Comment:
```suggestion
    <p>The Resource Description Framework (RDF) is a general-purpose framework for representing information on the Web. RDF-star extends RDF with a convenient way to make statements about other statements. This specification defines the abstract syntax of RDF-star as an extension of RDF's. It extends a number of RDF concrete syntaxes to support both the new abstract syntax and RDF's formal semantics. This specification also extends the SPARQL language to allow querying and updating of RDF-star data.</p>
```

Reply:
That's a misinterpretation of what I meant. "as well as" was about "it extends", not about "to support"... I'll rephrase that with shorter sentences (as I believe this is the essence of this suggestion), hopefully in a less ambiguous way.